packages/angular_devkit/core/src/experimental/jobs/strategy.ts

```diff
@@ -126,7 +125,7 @@ export namespace strategy {
 
     return (handler, options) => {
       const newHandler = (argument: A, context: JobHandlerContext<A, I, O>) => {
-        const argumentJson = stableStringify(argument);
+        const argumentJson = JSON.stringify(argument);
```

Comment:
Technically this would cause a miss for `{ a: 1, b: 2 }` and `{ b: 2, a: 1 }`.  However, this function currently is not used inside the CLI and experimental. Maybe a basic key sort first would be sufficient and capture the overwhelming majority of cases.